precompiled-token/src/getACAinfo.ts

```diff
@@ -0,0 +1,67 @@
+import { use } from 'chai';
+import { Contract } from 'ethers';
+import ADDRESS from "@acala-network/contracts/utils/Address"
+
+import { evmChai } from '@acala-network/bodhi';
+
+const Token = require('@acala-network/contracts/build/contracts/Token.json');
+import setup from './setup';
+
+use(evmChai);
+
+const main = async () => {
+    const { wallet, provider } = await setup();
+
+    console.log('Retrieve predeployed token information');
+
+    const instance = new Contract(ADDRESS.ACA, Token.abi, wallet);
+
+    console.log("Token address:", instance.address);
+
+    const name = await instance.name();
+    const symbol = await instance.symbol();
+    const decimals = await instance.decimals();
+    const totalSupply = await instance.totalSupply();
+    
+    const walletAddress = await wallet.getAddress();
+
+    const balance = await instance.balanceOf(await wallet.getAddress());
+
+    console.log("Token name:", name);
+    console.log("Token symbol:", symbol);
+    console.log("Token decimal spaces:", decimals);
+    console.log("Token total supply:", totalSupply.toString());
+    console.log("Token balance of %s is: %s", walletAddress, balance.toString());
+
+    const formattedSupply = balanceFormatting(totalSupply.toString(), decimals);
```

Comment:
`ethers` lib has a util to do this, reference: https://github.com/AcalaNetwork/bodhi.js/blob/master/eth-rpc-adapter/src/__tests__/e2e/endpoint.test.ts#L506

Reply:
It was a different util than this one, but it was a good suggestion. Thank you! :raised_hands: